cmake/Findelpp.cmake

```diff
@@ -2,11 +2,12 @@ set(ELPP_LIB_NAME "mapf::elpp")
 set(ELPP_INCLUDE_DIRS ${CMAKE_SOURCE_DIR}/framework/external/easylogging)
 set(ELPP_LIBRARY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/libelpp.so.1.4.0)
 
-add_definitions(-DELPP_THREAD_SAFE)
-add_definitions(-DELPP_FORCE_USE_STD_THREAD)
-add_definitions(-DELPP_NO_DEFAULT_LOG_FILE)
 add_library(${ELPP_LIB_NAME} UNKNOWN IMPORTED)
 
+set_target_properties(${ELPP_LIB_NAME} PROPERTIES
```

Comment:
This should be 2 separate commits - one which unifies the `add_definitions` into a single `set_target_properties`, and one commit which adds the `ELPP_SYSLOG`

Reply:
You're probably right, but since it doesn't change the previous behavior, I'll keep it as it